client/js/middleware/serialization/serializer_utils.js

```diff
@@ -0,0 +1,8 @@
+import _    from 'lodash';
+
+export function scrub(item) {
+  let scrubbedItem = _.cloneDeep(item);
+  scrubbedItem = _.omitBy(scrubbedItem, _.isNil);
+  scrubbedItem = _.omitBy(scrubbedItem, _.isEmpty);
```

Comment:
This won't allow us to send up either `""` or any number to qbank. I think this could be fixed by giving omitBy a predicate that checks to make sure the thing that `_.isEmpty` is called on is a collection.